Tnigam/VidamintToken
# Fixing tests

I Fixed:
1. VidaCrowdsale.js
2. VidaTokenTimeVault.js
3. VidacappedCrowdsale.js

I cannot fix vidaMigrate.js and i think the reason is because the zepplin Crowdsale contract has ownership of the token contract. Thereby making it the master of UpgradableToken. Which means only the crowdsale contract can call setUpgradeAgent(). I think the expectation is that `this.owner= '0xdf08f82de32b8d460adbe8d72043e3a7e25a3b39'` is suppose to be able to setUpgradeAgent.

Basically, because of the error above the beforeEach() will not complete. 

To Test:

1. fix migrate/2_migrate.js to work for tests
2. truffle test ./test/vidaMigrate.js

EXPECT:
1. A successful empty test

CURRENTLY:
1. We get an error -- > Contract: Crowdsale "before each" hook for "supply should be transferred":
                                       Error: VM Exception while processing transaction: invalid opcode

In the logs from the test, notice how `this.owner` and `token owner` are not the same. The address in `token owner` is the address of the `crowdsale address`

If you remove the `onlyMaster` modifier from the `setUpgradeAgent` function inside of `UpgradableToken.sol`, the above test will complete successfully. 

Code of interest:
Lines 49 and 58-60 inside of the open zepplin CrowdSale.sol contract.
